js/modules/networkgraph/draggable-nodes.js

```diff
@@ -1,37 +1,33 @@
 /* *
- * Networkgraph series
  *
- * (c) 2010-2019 Paweł Fus
+ *  Networkgraph series
```

Comment:
Actually, PackedBubble series also uses this module. Should move this to mixins or modules?